lib/knigge/implementation.ex

```diff
@@ -9,6 +9,12 @@ defmodule Knigge.Implementation do
 
   alias Knigge.Options
 
+  def fetch!(%Options{implementation: {:config, otp_app, [key | keys]}}) do
+    otp_app
+    |> Application.fetch_env!(key)
+    |> get_in(keys)
+  end
+
```

Comment:
Sadly, this doesn't really play nice with the `default` changes from #18.

It would also fail quietly when `key` exists but `get_in` returns `nil` for `keys`.

Instead I'd would expect to receive an error along the lines of:

> Unable to fetch implementation for `MyFacade` from config of app `<otp_app>` under path `<keys>`.

The message could be improved upon but I think you catch my drift?